backend/test.go

```diff
@@ -0,0 +1,51 @@
+package main
```

Comment:
I assume this was not supposed to be committed?

Reply:
oops, yes, commit by mistake )